models/base_sources/arbitrum_base_sources.yml

```diff
@@ -1,41 +1,143 @@
 version: 2
 
 sources:
-# Base Tables
+  # Base Tables
   - name: arbitrum
-    description: "Arbitrum raw tables"
+    description: "Arbitrum raw tables including transactions, traces and logs."
     freshness:
       warn_after: { count: 12, period: hour }
     tables:
+      - name: transactions_0006
```

Comment:
we shouldn't have a need for ```_000*``` tables in spellbook schema files / models

Reply:
any use of transactions will come out of ```transactions``` source

Reply:
should I update for ethereal as well?

Reply:
ethereum*

Reply:
fixing now.